packages/clerk-js/src/ui/customizables/__tests__/elementDescriptors.test.tsx

```diff
@@ -0,0 +1,118 @@
+// eslint-disable-next-line simple-import-sort/imports
+import { render, screen } from '../../../testUtils';
+import React from 'react';
+
+import { Box, descriptors } from '..';
+import { AppearanceProvider } from '../AppearanceContext';
+import { InternalThemeProvider } from '../../styledSystem';
+
+describe('Targetable classes', () => {
+  it('default stable class is added to the element ', () => {
+    render(
+      <AppearanceProvider appearanceKey='signIn'>
+        <Box
+          data-testid='test'
+          elementDescriptor={descriptors.socialButtonsIconButton}
+        />
+      </AppearanceProvider>,
+    );
+
+    expect(screen.getByTestId('test')).toHaveClass('cl-socialButtonsIconButton');
+  });
+
+  it('id related stable classname is added to the element', () => {
```

Comment:
🙃 We can test all the social providers with `it.each`

Reply:
Not sure if it matters here, it's not what this is supposed to be testing. The socialButtons was just a good example that has everything!